### PR TITLE
runtime: fix message which assumes the runtime is runc

### PR DESCRIPTION
**Pull request by giuseppe**
Signed-off-by: Giuseppe Scrivano <gscrivan@redhat.com>

### CODE REVIEW

File: libpod/runtime.go [hunk 1]
```diff
@@ -395,7 +395,7 @@ func makeRuntime(runtime *Runtime) (err error) {
 	}
 	if !foundRuntime {
 		return errors.Wrapf(ErrInvalidArg,
-			"could not find a working runc binary (configured options: %v)",
+			"could not find a working binary (configured options: %v)",
```
Comment (rhatdan) [inline libpod/runtime.go]: Don't we have a path to the runtime?

Comment (mheon) [reply libpod/runtime.go]: We get a bunch of candidate paths in the config file. This error happens if none of them exist.

**Comment by mheon** [comment]
This LGTM.

**Pull request merged**